src/bundle/Resources/views/themes/admin/content/warning_icon.html.twig

```diff
@@ -0,0 +1,5 @@
+<span class="ez-warning-icon">
```

Comment:
I'm not sure if this `/content` is wright place for the warning-icon.
Maybe `/ui/component/`?